### ISSUE TITLE
[ToggleGroup] Orientation prop has no effect

### ISSUE THREAD

**Issue by petegriffith** [radix-ui/primitives#1643]
### Current Behavior

The orientation prop has no effect on `ToggleGroup`. No matter what, the ToggleGroup has a horizontal orientation.

### Expected behavior

Similar to the Slider, `orientation` ought to change the ToggleGroup's orientation; otherwise it should be removed as an option

### Reproducible example

**Note** The following link comes directly from the ToggleGroup [docs](https://www.radix-ui.com/docs/primitives/components/toggle-group). I am also seeing this issue when running locally.

[CodeSandbox Template](https://codesandbox.io/s/kvb8uo?module=App.js&file=/App.js)

### Your environment

| Software         | Name(s) | Version |
| ---------------- | ------- | ------- |
| Radix Package(s) |     @radix-ui/react-toggle-group   |     1.0.0    |
| React            | n/a     |     18.0.0    |
| Browser          |  Chrome     |     105.0.5195.52    |
| Assistive tech   |         |         |
| Node             | n/a     |     16.15.0    |
| npm/yarn         |     pnpm    |  6.32.13       |
| Operating System |    macOS     |     Monterey 12.5.1     |

**Comment by jjenzz** [comment]
@benoitgrelard @andy-hook i wonder if it would be handy to update the docs surrounding this prop because the current docs aren't very clear here:

![CleanShot 2022-09-01 at 10 54 53](https://user-images.githubusercontent.com/175330/187886745-8c6144cb-cae5-4543-a899-10d533eb7644.png)

@petegriffith since radix is a style-free library, the `orientation` prop won't change the **visual** orientation (it is up to you to style it that way) but will change the **functional** orientation. in the case of `ToggleGroup`, vertical orientation will mean up/down arrow keys move focus within the component.

**Comment by benoitgrelard** [comment]
Yeah, @jjenzz is correct. I will rename and transfer this issue to the website repo instead for us to follow up on.
